geomstats/information_geometry/statistical_metric.py

```diff
@@ -422,3 +423,50 @@ def amari_divergence_tensor(self, base_point):
         first_primal = self.primal_connection.first_kind_christoffels(base_point)
         first_dual = self.dual_connection.first_kind_christoffels(base_point)
         return first_dual - first_primal
+
+class PotentialFunction:
+    def __init__(self, potential_function):
+        self.potential_function = potential_function
```

Comment:
I would make this private, as this is just a way of creating this object without having to inherit. An alternative would be to directly implement `PotentialFunction` as class and override `__call__` method.